devfiles/che4z/devfile.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: 1.0.0
+metadata:
+  generateName: che4z-
+components:
+  - 
+    type: chePlugin
+    id: broadcommfd/cobol-language-support/latest
+  - 
+    type: chePlugin
+    id: bitlang/cobol/latest
+  - 
+    type: chePlugin
+    id: broadcommfd/explorer-for-endevor/latest
+  - 
+    id: eclipse/che-machine-exec-plugin/7.3.0
+    type: chePlugin
+  - 
+    id: eclipse/che-theia/7.3.0
+    type: cheEditor
+  - 
+    alias: zowe-cli
+    type: kubernetes
+    mountSources: true
+    referenceContent: |
+         kind: List
+         items:
+          -
+           apiVersion: v1
+           kind: Pod
+           metadata:
+            name: zowe
+           spec:
+            volumes:
+             - name: zowe
+               persistentVolumeClaim:
+                 claimName: zowe-claim
+            containers:
+             -
+              image: 'ca-docker-brightside-docker.bintray.io/cli-mainframe-basic-stack:2019-11-19T15-56-35'
+              env:
+               - name: THEIA_PLUGINS
+                 value: local-dir:///plugins
```

Comment:
I'm not clear on why this env var is here, since this container does not mount `/plugins`

Reply:
THEIA_PLUGINS was set because the docker container also bundles a the Zowe Explorer VSCode extension, deployed via the theia-remote library. Longer term we would like to separate the VSCode extension into a separate plugin container and use `chePlugin`, but the first pass was bundled due to colocation of runtime and filesystem dependencies. The Zowe Explorer was built for a user workstation, and it's undergoing enhancements in parallel to better support Che+Theia deployment.

Reply:
Makes sense to me, so the Zowe Explorer runs in the `ca-docker-brightside-docker.bintray.io/cli-mainframe-basic-stack:2019-11-19T15-56-35` image instead of in the Theia container?

Reply:
That's correct